streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -678,48 +678,48 @@ public KTable<K, V> toTable(final Named named,
     }
 
     @Override
-    public <KR> KGroupedStream<KR, V> groupBy(final KeyValueMapper<? super K, ? super V, KR> keySelector) {
```

Comment:
Just re-ordering method to align to `KStream` interface order, plus updating `<KR>` to `<KOut>`